source/libs/page-detect.ts

```diff
@@ -407,3 +407,10 @@ export const hasCode = (): boolean => // Static code, not the editor
 	isGist() ||
 	isCompare() ||
 	isBlame();
+
+export const isActionPage = (): boolean => /\/marketplace\/actions\/.*/.test(location.pathname);
```

Comment:
Prefer `startsWith`


```suggestion
export const isActionPage = (): boolean => location.pathname.startsWith('/marketplace/actions/');
```

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-string-starts-ends-with.md

Reply:
Done, thanks.